Exercise_D and E/index.css

```diff
@@ -0,0 +1,87 @@
+nav {
+    height: 80px;
+    background: white;
+    padding: 0px 20px;
+    display: flex;
+    align-items: center;    
+}
+
+nav img{
+    height: 70px;
+}
+
+header {
+    background-image:    linear-gradient(
```

Comment:
Great use of linear gradient to make the image darker. 
Fantastic use of flex to position the h1 and button.